src/main/java/com/ericsson/ei/subscription/SubscriptionHandler.java

```diff
@@ -96,17 +99,23 @@ private void extractConditions(final String aggregatedObject,
             final JsonNode subscriptionJson = new ObjectMapper().readTree(
                     subscriptionData);
             LOGGER.debug("SubscriptionJson : " + subscriptionJson.toString());
-            LOGGER.debug("Aggregated Object : " + aggregatedObject);
+            LOGGER.debug("Aggregated Object : " + aggregatedObject +" for the id: " + id);
```

Comment:
missing a space between + and ""

Reply:
Also what does this log statement mean? Does the aggregated object have this id? Or is it an event id that has recently been processed?

Reply:
This is for the event id that was recently processed.